sysutils/pfSense-pkg-RRD_Summary/Makefile

```diff
@@ -1,8 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	pfSense-pkg-RRD_Summary
-PORTVERSION=	1.3.2
-PORTREVISION=	2
+PORTVERSION=	2.0
+PORTREVISION=	1
```

Comment:
When PORTVERSION changes, PORTREVISION line must be removed

Reply:
Done.